main/src/cgeo/geocaching/location/GeopointParser.java

```diff
@@ -74,6 +75,14 @@ public static Geopoint parse(@NonNull final String text) {
             // ignore and continue parsing textual formats
         }
 
+        // try to parse UTM string
+        try {
+            final UTMPoint utmPoint = new UTMPoint(text);
+            return utmPoint.toLatLong();
+        } catch (final Exception e) {
```

Comment:
We tend to call those exceptions `ignore` or `ignored` in new code because some tools get this as a hint that we do not expect to do anything with the exception.